pkgs/applications/networking/browsers/google-chrome/default.nix

```
@@ -39,6 +39,9 @@
 
 , gsettings-desktop-schemas
 , gnome3
+
+# For video acceleration under libva
+, libvaSupport ? true, libva ? null
```

```suggestion
, libvaSupport ? true, libva
```
the package should not be null.

---

Ok will update. I was following the pattern that I've above and elsewhere.

---

So... When should someone use `? null` vs not?

---

Might depend on the reviewer but AFAIK most are currently trying to avoid it whenever possible (i.e. almost always).